R/axis_geo.R

```diff
@@ -49,33 +47,39 @@
 #' are requested they will be added sequentially outwards starting from the plot
 #' border. The axis will always be placed on the outside of the last scale.
 #'
+#' If you would like to use intervals from the Geological Time Scale 2012
+#' (\code{\link{GTS2012}}), you can use \code{\link{get_timescale_data}} and
+#' supply the returned \code{data.frame} to the \code{intervals} argument.
+#'
 #' \code{axis_geo_phylo(...)} is shorthand for
 #' \code{axis_geo(..., phylo = TRUE)}.
 #'
 #' @param side \code{integer}. Which side to add the axis to (\code{1}: bottom,
 #'   the default; \code{2}: left; \code{3}: top; \code{4}: right).
 #' @param intervals The interval information to use to plot the axis: either A)
-#'   a \code{character} string indicating a built-in or remotely hosted
-#'   \code{data.frame} (see \code{\link[deeptime]{get_scale_data}}), or B) a
+#'   a \code{character} string indicating a rank of intervals from the built-in
+#'   \code{\link{GTS2020}}, B) a \code{character} string indicating a remotely
+#'   hosted \code{data.frame} (see \code{\link{get_timescale_data}}), or C) a
 #'   custom \code{data.frame} of time interval boundaries (see Details).
 #' @param height \code{numeric}. The relative height (or width if \code{side} is
 #'   \code{2} or \code{4}) of the scale. This is relative to the height (if
 #'   \code{side} is \code{1} or \code{3}) or width (if \code{side} is \code{2}
 #'   or \code{4}) of the plot.
 #' @param fill \code{character}. The fill color of the boxes. The default is to
```

Comment:
```suggestion
#' @param fill \code{character}. The fill colour of the boxes. The default is to
```
I think you're trying to standardise?